### PR TITLE
Feature/ws connect

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

**Comment by tejohnso** [comment]
Looks good to me.
@stulees I think you should take a look.

File: src-js/Viewer/LocalMessaging.js [hunk 1]
```diff
@@ -74,6 +81,11 @@ service.init = (embeddedViewer, preview, cb) => {
   connection.open();
 };
 
+service.sendFirstConnectionMessage = () => {
```
Comment (stulees) [inline src-js/Viewer/LocalMessaging.js]: @rodrigopavezi Does this function doesn't need to be surfaced in the API if it's being called from line 45 and isn't needed for test coverage?

**Comment by rodrigopavezi** [comment]
[Create LocalMessaging class on common-component lib. [2]](https://trello.com/c/B1UGUaLL/4702-create-localmessaging-class-on-common-component-lib-2)

**Comment by rodrigopavezi** [comment]
@stulees please review changes. cheers

**Pull request merged**